### PR TITLE
added some introduction to why we should be using the proper naming

### PR DESCRIPTION
**Pull request by sandrinethomas**

### CODE REVIEW

**Comment by sandrinethomas** [comment]
@womullan, I am not sure what is wrong with the checks... :/  Could you look into it?

File: index.rst [hunk 1]
```diff
@@ -6,20 +6,20 @@
 
 Abstract
 ========
+Using the proper naming for our Observatory and Telescope is important, as they both highlight important contribution to both the astronomical science and the construction of the telescope. The following page https://project.lsst.org/documents/name-use-guide summarizing the use of these names (and others) in the context of the publishing and presenting the work done on or with the Vera C. Rubin Observatory.
```
Comment (ranpalgill) [inline index.rst:9]: naming for our Observatory, Survey and Telescope

Comment (ranpalgill) [inline index.rst:9]: replace both with all

File: index.rst [hunk 2]
```diff
@@ -6,20 +6,20 @@
...
 ========
+Using the proper naming for our Observatory and Telescope is important, as they both highlight important contribution to both the astronomical science and the construction of the telescope. The following page https://project.lsst.org/documents/name-use-guide summarizing the use of these names (and others) in the context of the publishing and presenting the work done on or with the Vera C. Rubin Observatory. 
 
-Guidelines on how to use the name Simonyi Survey Telescope in the different Rubin Observatory products
+This document outlines the guidelines on how to specifically use the name Simonyi Survey Telescope within the different Rubin Observatory products, developed by the project.
 
 Usage of the Simonyi Name
 =========================
+As mentioned above, it is important to recognize properly the contribution of the Simonyi Family and proper usage of the Simonyi Survey Telescope denomination is one way to do so.
+In general, we do avoid the use abbreviations which loose the name we intend to honor.
```
Comment (ranpalgill) [inline index.rst:16]: In general, we avoid the use of abbreviations to prevent the loss of the name we intend to honor.

**Review by ranpalgill — COMMENTED** [review]
i think i'm done with my review. I think it would be nice to be able to attach a sound file so that people know how to pronounce Simonyi

**Pull request merged**